lib/rawQueries.js

```diff
@@ -9,12 +9,14 @@ const BuildFactoryQueries = {
         WHERE rank > :offset AND rank <= :maxRank
         ORDER BY "jobId", "id" ASC`,
 
-    statusesQueryMySql: `SELECT id, jobId, status, startTime, endTime
-        FROM (SELECT id, jobId, status, startTime, endTime,
-        RANK() OVER ( PARTITION BY jobId ORDER BY id DESC ) AS \`rank\`
-        FROM builds WHERE jobId in (:jobIds)) as R
-        WHERE \`rank\` > :offset AND \`rank\` <= :maxRank
-        ORDER BY jobId, id ASC`,
+    statusesQueryMySql: `SELECT a.id, a.jobId, a.status, a.startTime, a.endTime FROM (
```

Comment:
I am getting the error ``Unknown column 'a.id' in 'field list'``.
It seems necessary to change as follows.
`SELECT id, jobId, status, startTime, endTime`

Reply:
fixed